po/tr.po

```diff
@@ -67,245 +67,239 @@ msgstr "Merhaba Dünya!"
 msgid "Pattern Matching"
-msgstr "Şablon Eşleştirme"
+msgstr "Desen Eşleştirme"
 
-#: src/SUMMARY.md:66 src/pattern-matching/destructuring-enums.md:1
+#: src/SUMMARY.md:67 src/pattern-matching/destructuring-enums.md:1
 msgid "Destructuring Enums"
-msgstr "Enumları Yok Etme"
+msgstr "Enumların Çözümlenmesi"
 
-#: src/SUMMARY.md:67 src/pattern-matching/destructuring-structs.md:1
+#: src/SUMMARY.md:68 src/pattern-matching/destructuring-structs.md:1
 msgid "Destructuring Structs"
-msgstr "Veri Yapılarını Yok Etme"
+msgstr "Yapıların (Struct) Çözümlenmesi"
 
-#: src/SUMMARY.md:68 src/pattern-matching/destructuring-arrays.md:1
+#: src/SUMMARY.md:69 src/pattern-matching/destructuring-arrays.md:1
 msgid "Destructuring Arrays"
-msgstr "Dizileri Yok Etme"
+msgstr "Dizilerin Çözümlenmesi"
 
-#: src/SUMMARY.md:69 src/pattern-matching/match-guards.md:1
+#: src/SUMMARY.md:70 src/pattern-matching/match-guards.md:1
 msgid "Match Guards"
-msgstr "Eşleştirme Muhafızları"
+msgstr "Eşleşme Koşulları/Filtreleri"
 
-#: src/SUMMARY.md:72 src/exercises/day-1/luhn.md:1
+#: src/SUMMARY.md:73 src/exercises/day-1/luhn.md:1
 #: src/exercises/day-1/solutions-afternoon.md:3
 msgid "Luhn Algorithm"
 msgstr "Luhn Algrotiması"
 
-#: src/SUMMARY.md:73
-#, fuzzy
-msgid "Pattern Matching (TBD)"
-msgstr "Şablon Eşleştirme"
-
-#: src/SUMMARY.md:75
+#: src/SUMMARY.md:76
 msgid "Day 2: Morning"
 msgstr "Gün 2: Sabah"
 
-#: src/SUMMARY.md:81 src/memory-management.md:1
+#: src/SUMMARY.md:82 src/memory-management.md:1
 msgid "Memory Management"
 msgstr "Bellek Yönetimi"
 
-#: src/SUMMARY.md:82
+#: src/SUMMARY.md:83
 msgid "Stack vs Heap"
 msgstr "Stack vs Heap"
 
-#: src/SUMMARY.md:83
+#: src/SUMMARY.md:84
 msgid "Stack Memory"
 msgstr "Stack Belleği"
 
-#: src/SUMMARY.md:84 src/memory-management/manual.md:1
+#: src/SUMMARY.md:85 src/memory-management/manual.md:1
 msgid "Manual Memory Management"
 msgstr "Manuel Bellek Yönetimi"
 
-#: src/SUMMARY.md:85 src/memory-management/scope-based.md:1
+#: src/SUMMARY.md:86 src/memory-management/scope-based.md:1
 msgid "Scope-Based Memory Management"
 msgstr "Kapsam Tabanlı Bellek Yönetimi"
 
-#: src/SUMMARY.md:86
+#: src/SUMMARY.md:87
 msgid "Garbage Collection"
 msgstr "Çöp Toplama"
 
-#: src/SUMMARY.md:87
+#: src/SUMMARY.md:88
 msgid "Rust Memory Management"
 msgstr "Rust Bellek Yönetimi"
 
-#: src/SUMMARY.md:88 src/memory-management/comparison.md:1
-msgid "Comparison"
-msgstr "Karşılaştırma"
-
 #: src/SUMMARY.md:89 src/ownership.md:1
 msgid "Ownership"
 msgstr "Sahiplik"
 
 #: src/SUMMARY.md:90 src/ownership/move-semantics.md:1
 msgid "Move Semantics"
-msgstr "Taşıma Şematiği"
+msgstr "Taşıma Semantiği"
 
 #: src/SUMMARY.md:91 src/ownership/moved-strings-rust.md:1
 msgid "Moved Strings in Rust"
 msgstr "Rust'da Taşınmış Stringler"
 
 #: src/SUMMARY.md:92
 msgid "Double Frees in Modern C++"
-msgstr "Modern C++'da Çifte Serbestlik"
+msgstr "Modern C++'da Double Free'ler (Adresi iki kez serbest bırakma)"
```

Comment:
Hi, 
In [the english copy](https://google.github.io/comprehensive-rust/ownership/double-free-modern-cpp.html), this has been also translated as "Defensive Copiers in Modern c++". 
Do you think that may be a more helpful phrasing for the Turkish language reader?

Reply:
Hi, No I don't think so. While translating it into Turkish, I try to find a word that is well-established and that will mean something. Otherwise, it is whatever is in use.